lib/common.sh

```diff
@@ -178,9 +178,13 @@ else
   echo "Management cluster forced to be minikube when container runtime is not docker"
   export EPHEMERAL_CLUSTER="minikube"
 fi
+
 #Kustomize version
 export KUSTOMIZE_VERSION=${KUSTOMIZE_VERSION:-"v3.6.1"}
 
+#clusterctl version
+export CLUSTERCTL_VERSION=${CLUSTERCTL_VERSION:-"v0.3.8"}
```

Comment:
You should probably use the CAPIRELEASE variable to get the latest CAPI release (you need to import lib/releases.sh)